.github/workflows/terraform-apply-env.yml

```diff
@@ -26,6 +26,8 @@ jobs:
       TERRAFORM_PRE_RUN: |
           apt-get update
           apt-get install -y zip python
+      TF_VAR_postgrest_image: 'ghcr.io/gsa-tts/fac/postgrest:latest'
```

Comment:
For some reason, we never had the postgrest or clamav image explicitly defined inside `terraform-apply-env.yml`. Since `preview` will not be using any prs, and instead will have a `workflow_dispatch` called from any branch, this should be here. Plus, it should be here regardless.